src/components/shop/confirm.js

```diff
@@ -58,9 +58,15 @@ export const createConfirm = scene => {
     container.setBalance = bal => balance.setText(bal);
     container.getBalance = () => balance.getValue();
     container.setLegal = setLegal(container);
+    container.getTextElems = () =>
+        [container.elems.prompt, container.elems.price, ...container.elems.item].filter(isText);
```

Comment:
Why not just return all `container.elems` instead of destructuring specific ones?

Reply:
Just thinking this might make it a bit more future-proof...

Reply:
There needs to be a bit of destructuring as some of the elems are single text / image elems, but the item is an array of elems (since we can have a couple of different views on the item.) Can simplify it though- doing that.